README.md

```diff
@@ -16,19 +18,22 @@ go get -u github.com/ncr4/tuneuptechnology-go
 package main
 
 import (
+	"fmt"
+	"os"
+	"encoding/json"
 	"github.com/ncr4/tuneuptechnology-go"
 )
 
 func main() {
-	tuneuptechnology.CreateCustomer(
-		// Pass in your email and API key
-		&tuneuptechnology.Client{
-			Auth: "",
-			APIKey: "",
-		},
+	// Setup your email and API key
+	api_email := "EMAIL_HERE"
+	api_key := "API_KEY_HERE"
```

Comment:
use Environment variables here instead of hard coding values
```suggestion
	api_email := os.Getenv("ACCOUNT_EMAIL")
	api_key := os.Getenv("ACCOUNT_KEY")
```

Reply:
Also, consider adding support for a `.env` file that you can drop in as well

Reply:
Good point